pages/privacy.tsx

```diff
@@ -86,7 +86,7 @@ export default function Home({ copyrightYear }: InferGetServerSidePropsType<type
           <h2>5. Rekisterin tietosisältö</h2>
           <p>
             <IoLogoGithub />
-            <a href="https://github.com/Testausserveri/testausapis/blob/715fe1fa313327124909aa94c16bcc3b059911da/src/database/schemas/userInfo.js">Rekisterin tietokantarakenne on avointa lähdekoodia ja tarkasteltavissa myös GitHubissa</a>.
+            <a href="https://github.com/Testausserveri/testausapis/blob/main/src/database/schemas/userInfo.js">Rekisterin tietokantarakenne on avointa lähdekoodia ja tarkasteltavissa myös GitHubissa</a>.
```

Comment:
Should this instead be pinned to a single commit, in case that file is moved/deleted later?

Reply:
I don't think so. If the file is moved and the link is incorrect, which probably means code has changed, the user should get a 404 and not incorrect information.

Reply:
Yes that's what I thought as well. Currently it linked to an old version so I changed it to that.